practice-one/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = (props) => {
+  const {
+    type,
+    children,
+    width,
+    height,
+    color,
+    background,
+    fontSize,
+    fontWeight,
+    icon,
+    border,
+    align,
+  } = props;
+  const buttonClasses = `btn ${color} ${background} ${border} ${fontSize} ${fontWeight} ${align}`;
+  const buttonSizes = {
+    "--btn-width": width,
+    "--btn-height": height,
+  };
+  return (
+    <button
+      type={type ? type : "button"}
```

Comment:
can use default type = "button", no need to do this condition

Reply:
I fixed 
![image](https://github.com/datdragon777/React-Training/assets/74806826/a751a0bc-37c3-465c-b850-9f22c331a776)

Reply:
did you understand my mean?